source/includes/warning-tools-dollar-import-export-restriction.rst

```diff
@@ -0,0 +1,14 @@
+.. warning:: Data Dump and Restore Conflicts with $ Prefix in Fields
+
+   Starting in MongoDB 5.0, document field names can be prefixed with a
+   dollar character (``$``). However, :binary:`~bin.mongoimport` and
+   :binary:`~bin.mongoexport` won't work with field names that are prefixed
+   with a dollar character.
```

Comment:
We should also be specific that mongodump and mongorestore won't work if $-prefixed field is in a collection's options.

Reply:
Done